pkg/skaffold/yamltags/tags_test.go

```diff
@@ -277,10 +277,33 @@ func TestGetYamlTags(t *testing.T) {
 			expectedTags: []string{"c"},
 		},
 		{
-			name:         "non-pointer fields are returned in empty struct",
-			yaml:         struct{ A string }{},
+			name: "non-pointer fields are returned in empty struct",
+			yaml: struct {
+				A string `yaml:"a"`
+			}{},
 			expectedTags: []string{"a"},
 		},
+		{
+			name: "no yaml tag",
+			yaml: struct {
+				Foo string
+			}{Foo: "bar"},
+			expectedTags: []string{"foo"},
+		},
+		{
+			name: "yaml flag `inline`",
+			yaml: struct {
+				Foo string `yaml:",inline"`
+			}{Foo: "bar"},
+			expectedTags: nil,
+		},
+		{
+			name: "empty yaml key",
+			yaml: struct {
+				Foo string `yaml:",omitempty"`
+			}{Foo: "bar"},
+			expectedTags: []string{"foo"},
```

Comment:
Would be helpful if this example used multi-word field to verify that just the first letter is lowercased:
```suggestion
				FooBar string `yaml:",omitempty"`
			}{FooBar: "bar"},
			expectedTags: []string{"fooBar"},
```